documentation/keys.md

```diff
@@ -0,0 +1,79 @@
+# Keys
+A detailed description of each key is listed below. These keys will be listed in the order they appear on the
+settings page. The "Alternate Functionality" listed below a feature can be activated by holding **SHIFT +**
+the key of the feature.
+## Move
+When looking at a region page, pressing this key will move your nation to the region.
+
+When looking at the reports page,
+pressing this key will refresh the page and click the region the raiders moved to, if applicable.
+## Endorse
+When looking at a nation page, pressing this key will endorse that nation, if possible.
+### Alternate Functionality
+When looking at an ajax2 activity feed, pressing this key will load in a list of nations to endorse.
+
+Otherwise, pressing this key will attempt to endorse the current nation you're looking at, or load the next nation
+to be endorsed if it has already been endorsed.
+## Refresh
+Pressing this key will refresh the current page.
+
+This feature is identical to the Breeze++ feature.
+## Dossier
+When looking at a nation page, pressing this key will dossier that nation, if possible.
+### Alternate Functionality
+When looking at an ajax2 activity feed, pressing this key will load in a list of nations to dossier.
+
+Otherwise, pressing this key will attempt to dossier the current nation you're looking at, or load the next nation
+to be dossed if it has already been dossed.
+## Move Back to Jump Point
+When looking at the region which has the same name as the **jump point you have in settings**, move to that region.
+
+Otherwise, open up the jump point region page.
+
+This feature is identical to the Breeze++ feature.
+## Endorse WA Delegate
+When looking at a **region** page, open up the nation page of its WA delegate.
+
+When looking at a **nation** page, endorse that nation.
+
+This feature is identical to the Breeze++ feature.
+## Check If Your Nation Updated
+Pressing this key will open up the minimal activity feed of the currently logged in nation. This can be used
+to check if your nation updated.
+
+This feature is identical to the Breeze++ feature.
+## Toggle Template
+When looking at a page that has a template, load the `template-overall=none` version of that page.
+
+Otherwise, load the templated version of that page.
+
+This feature is identical to the Breeze++ feature.
+## Open Activity Page
+Pressing this key will open up the world activity feed filtered to `member`, `move`, and `endo`. This page
+can be used to spot for raider activity.
+
+This feature is identical to the Breeze++ feature.
+## Open Reports Page
+Pressing this key will open up the Reports page.
+
+This feature is identical to the Breeze++ feature.
```

Comment:
Pressing spacebar twice with Breeze++ opens up a world Ajax2 activity page, which isn't something that Gauntlet does.